src/Management/test/Endpoint.Test/ManagementPort/ManagementEndpointServedOnDifferentPortTest.cs

```diff
@@ -98,6 +98,37 @@ public async Task AspNetDefaultPort_AlternateManagementPortConfigured_Accessible
         actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
     }
 
+    [Fact]
+    public async Task AspNetDefaultPort_AlternateManagementPortConfigured_AccessibleOnCfInstancePorts()
+    {
+        using var instancePorts = new EnvironmentVariableScope("CF_INSTANCE_PORTS",
+            "[{\"external\":40041,\"internal\":8090,\"external_tls_proxy\":8998,\"internal_tls_proxy\":8999}]");
+
+        const string managementPort = "8090";
+        const string externalTlsProxyPort = "8998";
+        const string internalTlsProxyPort = "8999";
```

Comment:
If these moved to the top of the method, they could be used in the `EnvironmentVariableScope` constructor.